### ISSUE TITLE
A router for the library

### ISSUE THREAD

**Issue by AlidotSal** [endorphinjs/endorphin#86]
What's your suggestion for routing? Is there any router to use with EndorphinJS?

**Comment by AlidotSal** [comment]
And also is there a concept of self in Endorphin? Similar to calling a component inside itself in other frameworks.

**Comment by sergeche** [comment]
Yes, you can use `<e:self />` tag to reference current component
